packages/bar/index.d.ts

```diff
@@ -153,7 +151,6 @@ declare module '@nivo/bar' {
             onClick: BarMouseEventHandler<SVGRectElement>
             onMouseEnter: BarMouseEventHandler<SVGRectElement>
             onMouseLeave: BarMouseEventHandler<SVGRectElement>
-            role: string
```

Comment:
I think you need to revert this deletion, but everything else looks good.

Reply:
Hmm, this might be my first accidental deletion. + I committed it 😅 shameful.
I am sorry for that, thanks for reviewing - I will resolve this soon

Reply:
No worries and no need to apologize! 😀